cmd/integration_test/build/buildspecs/test-eks-a-cli.yml

```diff
@@ -49,6 +49,9 @@ env:
     T_HTTP_PROXY: "proxy-config-data:httpProxy"
     T_HTTPS_PROXY: "proxy-config-data:httpsProxy"
     T_NO_PROXY: "proxy-config-data:noProxy"
+    T_HTTP_PROXY_CLOUDSTACK: "proxy-config-data:httpProxyCloudStack"
+    T_HTTPS_PROXY_CLOUDSTACK: "proxy-config-data:httpsProxyCloudStack"
+    T_NO_PROXY_CLOUDSTACK: "proxy-config-data:noProxyCloudStack"
```

Comment:
These fields should be added to secrets manager but I'm not sure how that's done. I could use some guidance for achieving this

Reply:
do you think we could reuse T_HTTP_PROXY, T_HTTPS_PROXY and T_NO_PROXY?

Reply:
Using the same proxy would be ideal, if that's possible.

Reply:
Values added to secrets manager by @abhay-krishna

Reply:
I am assuming we don't have proxy tests for docker then, because right now those env vars assume proxy is for vsphere, so maybe need to go with this approach